README.md

```diff
@@ -545,16 +545,19 @@ There are now three options for enabling docker DNS resolution from within the w
 
 2. If `LOCAL_TLD` is set (e.g. to local) write `/etc/dnsmasq.conf` to use `127.0.0.11` for that TLD. Also, write `/etc/resolv.conf` to search `LOCAL_TLD` so that containers can access the addresses of the services without having to know to append `.local` to match the rule in `/etc/dnsmasq.conf`. This will require aliases with the TLD in each of the containers that need to be accessible from within the wireguard network. An alias should look something like this (e.g. for `LOCAL_TLD=ghost`).
 
-   ```yaml
-   filebot:
-   ---
-   networks:
-     default:
-       aliases:
-         - filebot.ghost
-   ```
-
-   which will result in `filebot` being accessible from containers within the `wireguard` network.
+    ```yaml
+    filebot:
+      image: ghostserverd/filebot:4.9.x
+        container_name: filebot
+        restart: always
+        networks:
```

Comment:
I think you've got an extra `networks:` in here. it should look more like this

````
```yaml
filebot:
  image: ghostserverd/filebot:4.9.x
  container_name: filebot
  restart: always
  networks:
    default:
      aliases:
        - filebot.ghost
```
````